ledfx/effects/random.py

```diff
@@ -0,0 +1,92 @@
+import random
+import time
```

Comment:
This is what is making your precommit checks fail. 

remove time reference

However, I suspect you should add timeit and implement time invariance.